### ISSUE TITLE
Characters after " orange in StringReplace

### ISSUE THREAD

**Issue by TrueMilli** [mark-wiemer/vscode-autohotkey-plus-plus#262]
## Code snippet

```
StringReplace, Clipboard, Clipboard, ", , All
```

## Expected behavior

Characters after `"` should have the usual color in `StringReplace`.

## Editor tokens and scopes

"1 char
language	ahk
standard token type	String
foreground	#CE9178
background	#1E1E1E
contrast ratio	6.31
textmate scopes	punctuation.definition.string.begin.ahk
string.quoted.double.ahk
source.ahk
foreground	string
{ "foreground": "#CE9178" }

## Additional context

`"` does not start a string in this case but is a character to be replaced.

![image](https://user-images.githubusercontent.com/61841994/200139842-a218af46-842f-4429-bce1-249a496fba9d.png)

**Comment by fade2gray** [comment]
Incorrect syntax used; the single double-quote needs to be escaped as `""""`.

i.e. `StringReplace, Clipboard, Clipboard, """", , All`

...but, `StringReplace` has been [deprecated](https://www.autohotkey.com/docs/commands/StringReplace.htm), and you should be using `StrReplace`

i.e. `Clipboard := StrReplace(Clipboard, """")`

**Comment by mark-wiemer** [comment]
@fade2gray, when you say the syntax is incorrect, are you implying the code won't compile or work at all?

**Comment by fade2gray** [comment]
No, maybe I used the wrong terminology.

I was just trying to show that the replacement string in the given snippet is incorrectly formatted, and the resulting syntax highlighting gives you a clue that something's amiss.

![Capture4](https://user-images.githubusercontent.com/1332009/200165885-2e0bd6f1-c8b0-4d0d-b9c4-ea395bbd0d87.PNG)

**Comment by kyklish** [comment]
> No, maybe I used the wrong terminology.
> 
> I was just trying to show that the replacement string in the given snippet is incorrectly formatted, and the resulting syntax highlighting gives you a clue that something's amiss.
> 
> ![Capture4](https://user-images.githubusercontent.com/1332009/200165885-2e0bd6f1-c8b0-4d0d-b9c4-ea395bbd0d87.PNG)

No, it's correctly formatted. Inside commands AHK use `legacy text`, not `expression text`!
`StringReplace, Clipboard, Clipboard, ", , All` <-- correct
`StringReplace, Clipboard, Clipboard, % """", , All` <-- correct, `%` sign force to use `expression text`

**Comment by fade2gray** [comment]
Yep, my bad.

**Comment by kyklish** [comment]
> Yep, my bad.

My bad too, I edited my comment above.

> Within an expression, two consecutive quotes enclosed inside a literal string resolve to a single literal quote.
> For example: `Var := "The color ""red"" was found."`.
